rastervision/data/vector_source/vector_tile_vector_source.py

```diff
@@ -45,23 +43,43 @@ def process_features(features, map_extent, id_field):
             id_geoms.append(g)
 
         union_geom = cascaded_union(id_geoms)
-        geom = union_geom.intersection(extent_geom)
         union_feature = copy.deepcopy(id_features[0])
-        union_feature['geometry'] = mapping(geom)
+        union_feature['geometry'] = mapping(union_geom)
         proc_features.append(union_feature)
-    return proc_features
 
+    return {'type': 'FeatureCollection', 'features': proc_features}
 
-def vector_tile_to_geojson(uri, zoom, id_field, crs_transformer, extent):
-    """Get GeoJSON covering an extent from a vector tile endpoint.
 
-    Merges features that are split across tiles and crops against the extentself.
-    """
+tile_feature_cache = {}
```

Comment:
Is there a replacement policy?  Are there concerns about scalability (this is a question not a loaded question)?

Reply:
I'm not sure, but it's a better idea to set some limit on the size of the cache. Python 3 has a nice `lru_cache` callback that I'm using now. I'm not sure what a good limit size is for the cache, but 32 seems safe.

Reply:
> I'm not sure, but it's a better idea to set some limit on the size of the cache. Python 3 has a nice `lru_cache` callback that I'm using now. I'm not sure what a good limit size is for the cache, but 32 seems safe.

That sounds like a fine plan.